src/span-data.ts

```diff
@@ -45,7 +46,7 @@ interface StackFrame {
 // Auto-incrementing integer
 let uid = 1;
 
-export class SpanData {
+export class SpanData implements ISpanData {
```

Comment:
See 'Naming Style' section of the style guide on `I` prefix for interfaces. Disallowed.

Also two different files exporting the same type name is not great. Use different names.

Reply:
The externally facing types in `plugin-interface` should take precedence when it comes to name brevity; I don't want to call them `TraceAgentInterface`, etc.

I could follow in the footsteps of `grpc` and name the classes `*Impl*, but I'd want to do that as a separate change. For now, I hope it's acceptable to have `SpanDataInterface`?